packages/nextra-theme-blog/src/mdx-theme.tsx

```diff
@@ -40,9 +40,9 @@ const createHeaderLink =
   (Tag: `h${2 | 3 | 4 | 5 | 6}`) =>
   ({ children, id, ...props }: ComponentProps<'h2'>): ReactElement => {
     return (
-      <Tag {...props}>
+      <Tag className={`subheading-${Tag}`} {...props}>
         <span className="subheading-anchor -mt-8" id={id} />
-        <a href={`#${id}`}>
+        <a className="subheading-title" href={`#${id}`}>
```

Comment:
```suggestion
        <a href={`#${id}`}>
```
You can style this element by `.subheading-anchor + a`